presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/QueryBuilder.java

```diff
@@ -131,7 +133,9 @@ public PreparedStatement buildSql(
                     .append(Joiner.on(" AND ").join(clauses));
         }
 
-        PreparedStatement statement = client.getPreparedStatement(connection, sql.toString());
+        String query = sqlFunction.apply(sql.toString());
+
```

Comment:
nit: remove the empty line here

Reply:
That was intentional to separate the limit application from building the `PreparedStatement`.

Reply:
But I'll remove it since it looks fine without it